src/xdocs/config_filters.xml

```diff
@@ -973,6 +973,143 @@ public class InputTest {
 &lt;suppress-xpath checks=&quot;.*&quot; query=&quot;//METHOD_DEF[@text='legacyMethod']
           /descendant-or-self::node()&quot;/&gt;
         </source>
+        <p>
+          Currently, xpath queries support two types of attributes <code>@text</code> and
+          <code>@value</code>
+          <ul>
+            <li>
+              <code>@text</code> - addresses to the name of the node. For example: class name,
+              method name, variable name and etc.
+            </li>
+            <li>
+              <code>@value</code> - addresses to the value of the node. For example: variable value,
+              annotation value, text content and etc.
+            </li>
+          </ul>
+        </p>
+        <p>For example, to suppress violation on variable <code>wordCount</code> in following
+        code:</p>
+        <source>
+public class InputTest {
+    private int wordCount = 11;
+}
+        </source>
+        <p>You need to look at AST of such code by our CLI tool:</p>
+        <source>
+            $ java -jar checkstyle-X.XX-all.jar -t InputTest.java
```

Comment:
please remove leading spaces, it will generate more compact html.
Pleas apply to all other `source` tags.

Reply:
done